package.json

```diff
@@ -14,6 +14,7 @@
     "pretest": "sequelize db:migrate:undo:all --env test && sequelize db:migrate --env test",
     "test": "cross-env process.env.NODE_ENV=test nyc mocha --compilers js:babel-core/register ./server/tests/**/'*.test.js' --recursive --exit || true",
     "dev": "cross-env process.env.NODE_ENV=development npm run migrate && npm run eslint && npm run babel-node server/index.js",
+    "model-create": "sequelize model:create --name Followings --attributes follower:integer,followedBy:integer",
```

Comment:
Add this script to make model creating easy
just add the model name and its attributes